client/css/photos.css

```diff
@@ -20,10 +20,10 @@
     margin: 40px;
 }
 
-.picture-frame > .user-photo:active {
+/* .picture-frame > .user-photo:active {
```

Comment:
You can just take this out. I was playing around with photo enlarging and couldn't get it to fire right.